src/cli/README_DATASET.md

```diff
@@ -109,26 +109,36 @@ After the device successfully attaches to a Thread network, the device will retr
 
 ## Command List
 
-- [help](#help)
-- [active](#active)
-- [activetimestamp](#activetimestamp)
-- [channel](#channel)
-- [channelmask](#channelmask)
-- [clear](#clear)
-- [commit](#commit)
-- [delay](#delay)
-- [extpanid](#extpanid)
-- [init](#init)
-- [masterkey](#masterkey)
-- [meshlocalprefix](#meshlocalprefix)
-- [mgmtgetcommand](#mgmtgetcommand)
-- [mgmtsetcommand](#mgmtsetcommand)
-- [networkname](#networkname)
-- [panid](#panid)
-- [pending](#pending)
-- [pendingtimestamp](#pendingtimestamp)
-- [pskc](#pskc)
-- [securitypolicy](#securitypolicy)
+- [OpenThread CLI - Operational Datasets](#openthread-cli---operational-datasets)
+  - [Overview](#overview)
+    - [Active Operational Dataset](#active-operational-dataset)
+    - [Pending Operational Dataset](#pending-operational-dataset)
+  - [Quick Start](#quick-start)
+    - [Form Network](#form-network)
+    - [Attach to Existing Network](#attach-to-existing-network)
+  - [Command List](#command-list)
+  - [Command Details](#command-details)
+    - [help](#help)
+    - [active](#active)
+    - [activetimestamp](#activetimestamp)
+    - [channel](#channel)
+    - [channelmask](#channelmask)
+    - [clear](#clear)
+    - [commit](#commit)
+    - [delay](#delay)
+    - [extpanid](#extpanid)
+    - [init](#init)
+    - [masterkey](#masterkey)
+    - [meshlocalprefix](#meshlocalprefix)
+    - [mgmtgetcommand](#mgmtgetcommand)
+    - [mgmtsetcommand](#mgmtsetcommand)
+    - [networkname](#networkname)
+    - [panid](#panid)
+    - [pending](#pending)
+    - [pendingtimestamp](#pendingtimestamp)
+    - [pskc](#pskc)
+    - [securitypolicy](#securitypolicy)
```

Comment:
How about updating the [Security Policy](https://github.com/openthread/openthread/blob/main/src/cli/README_DATASET.md#securitypolicy) section?